php-checks/src/test/resources/checks/MethodNameReturningBooleanCheck.php

```diff
@@ -5,7 +5,7 @@ class A {
   /**
```

Comment:
@vilchik-elena It may have been worth adding some tests with comments like `@return int`...